www/js/banidb/realm-search.js

```diff
@@ -161,6 +161,30 @@ const loadShabad = ShabadID => (
   })
 );
 
+/**
+ * Retrieve all lines from a Ceremony
+ *
+ * @param {number} CermonyID The specific Shabad to get
+ * @returns {object} Returns array of objects for each line
+ * @example
+ *
+ * loadCeremony(3);
+ * // => [{ Gurmukhi: 'jo gurisK guru syvdy sy puMn prwxI ]', ID: 31057 },...]
+ */
+
+const loadCeremony = ceremonyID => (
+  new Promise((resolve, reject) => {
+    Realm.open(realmConfig)
```

Comment:
Could you please do the check to see if it's been initialized first?

Reply:
Sure. Sorry, missed it 🙈